descriptors/ModuleDescriptor-template.json

```diff
@@ -352,7 +352,12 @@
           "pathPattern": "/finance/allocations",
           "permissionsRequired": ["finance.allocations.item.post"],
           "modulePermissions": [
-            "finance-storage.transactions.item.post"
+            "finance-storage.transactions.item.post",
+            "finance-storage.funds.item.get",
+            "finance-storage.ledgers.item.get",
```

Comment:
I don't see any calls except get fund by id. No need  for following modulesPermissions
```
"finance-storage.fiscal-years.item.get"
 "finance-storage.fiscal-years.collection.get"
"finance-storage.group-fund-fiscal-years.collection.get"
```

Reply:
removed